packages/components/layout/src/Layout.tsx

```diff
@@ -73,9 +74,15 @@ const _Layout = (props: LayoutProps, ref: Ref<HTMLDivElement>) => {
           flexGrow={1}
           testId={contentTestId}
         >
-          {leftSidebar}
+          {React.isValidElement(leftSidebar) &&
+            React.cloneElement(leftSidebar, {
```

Comment:
maybe there's an easier way to pass a prop to a component that is itself passed as a prop?